src/BuildScriptGenerator/Node/NodeLanguageDetector.cs

```diff
@@ -149,5 +153,23 @@ private string DetectNodeVersion(dynamic packageJson)
 
             return nodeVersion;
         }
+
+        private bool IsStaticSite(ISourceRepo sourceRepo)
+        {
+            return IsHugoSite(sourceRepo);
+        }
+
+        private bool IsHugoSite(ISourceRepo sourceRepo)
+        {
+            if (sourceRepo.FileExists(NodeConstants.HugoJsonFileName) ||
+                sourceRepo.FileExists(NodeConstants.HugoTomlFileName) ||
+                sourceRepo.FileExists(NodeConstants.HugoYamlFileName) ||
+                sourceRepo.DirExists(NodeConstants.HugoConfigFolderName))
```

Comment:
I updated the logic in this block: we check for `config.toml` at the root, a `config` folder with a `config.toml` file in it, or a `config` folder with a `*.toml` file anywhere in it. I scrapped the checks for `config.json` and `config.yml` since they don't seem to be the streamlined scenarios (I couldn't find any examples online of apps being built with these two files), so I stuck with the other scenarios outlined in the [Hugo configuration doc](https://gohugo.io/getting-started/configuration/). Let me know what you think.